Makefile

```diff
@@ -1,13 +1,15 @@
 include Makefile.defs
 
 SUBDIRS = envoy plugins bpf cilium daemon monitor cilium-health bugtool
-GOFILES ?= $(shell go list ./... | grep -v /vendor/ | grep -v /contrib/ | grep -v /test | grep -v cilium/envoy | grep -v envoy.*api)
+GOFILES ?= $(shell go list ./... | grep -v /vendor/ | grep -v /contrib/ | grep -v /test | grep -v envoy.*api)
 GOLANGVERSION = $(shell go version 2>/dev/null | grep -Eo '(go[0-9].[0-9])')
-GOLANG_SRCFILES=$(shell for pkg in $GOFILES; do find $(pkg) -name *.go -print; done | grep -v /vendor/)
-BPF_SRCFILES=$(shell find bpf/ -name *.[ch] -print)
+GOLANG_SRCFILES=$(shell for pkg in $(subst github.com/cilium/cilium/,,$(GOFILES)); do find $$pkg -name *.go -print; done | grep -v /vendor/)
+BPF_SRCFILES=$(shell find bpf -name *.[ch] -print)
```

Comment:
Probably out of scope for this PR but this might be more reliable using BPF_FILES from https://github.com/cilium/cilium/blob/master/daemon/bpf.sha#L3

Reply:
Done!